back/users/serverless.yml

```diff
@@ -44,6 +46,8 @@ functions:
       USERS_TABLE_NAME: ${self:custom.usersTableName}
       IDEAS_TABLE_NAME: ${self:custom.ideasTableName}
       IDEA_COMMENTS_TEMPORARY_TABLE_NAME: ${self:custom.ideaCommentsTemporaryTableName}
+      AVATAR_STATIC_BUCKET_URL: ${self:provider.avatarsStaticBucketURL}
+      AVATAR_STATIC_BUCKET_NAME: ${self:provider.avatarsStaticBucketName}
```

Comment:
add to wercker please if u didnt do that already

Reply:
Does wercker need it? This is serverless config, something that handled on AWS.

Reply:
nvm missed that its filled from provider